src/table/components/TableHeadWrapper.jsx

```diff
@@ -2,13 +2,14 @@ import React, { memo, useMemo } from 'react';
 import PropTypes from 'prop-types';
 import { styled } from '@mui/system';
 import TableCell from '@mui/material/TableCell';
-import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
 import TableSortLabel from '@mui/material/TableSortLabel';
+import { TableHead } from '@mui/material';
```

Comment:
@LiKang6688 I think we should get the component explicitly from material to not load everything, do you remember if that is why we do that?